pkg/version/version.go

```diff
@@ -43,10 +41,7 @@ var (
 	catalogImage string //nolint:gochecknoglobals
 )
 
-// CatalogImage returns a catalog image needed for the build of everestctl
```

Comment:
Even though the rest of the comment is wrong, this specific line should not be removed

Reply:
And it's not, it can be found below :) for some reason the diff tool considered the string as fully removed and added below